packages/react-core/src/components/TreeView/examples/TreeView.md

```diff
@@ -521,11 +521,11 @@ class CheckboxTreeView extends React.Component {
     };
 
     this.mapTree = item => {
-      const hasCheck = areAllDescendantsChecked(item);
+      const hasCheckbox = areAllDescendantsChecked(item);
       // Reset checked properties to be updated
       item.checkProps.checked = false;
 
-      if (hasCheck) {
+      if (hasCheckbox) {
```

Comment:
Similar to a previous comment, I feel like this can remain as `hasCheck` since it's checking whether something is checked/has that checkmark.